api/types/types.proto

```diff
@@ -836,24 +836,23 @@ message Namespace {
 // NamespaceSpec is a namespace specificateion
 message NamespaceSpec {}
 
-message ResetPasswordTokenV3 {
+message UserTokenV3 {
     option (gogoproto.goproto_stringer) = false;
     option (gogoproto.stringer) = false;
 
     // Kind is a resource kind
```

Comment:
FYI: this is out of scope for your MR, but I think it would be nice to have some examples within the documentation. Ie, what is usually set in the `Kind` field? What about `SubKind`? Is `Version` a plain number, a semantic version, something else?

Reply:
I will leave theses as is, b/c all resource templates in teleport follow the same templating with `kind`, `subkind`, and `version`, etc.